examples/irsa/README.md

```diff
@@ -0,0 +1,70 @@
+# IAM Roles for Service Accounts
+
+This example shows how to create an IAM role to be used for a Kubernetes `ServiceAccount`. It will create a policy and role to be used by the [cluster-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) using the [public Helm chart](https://github.com/helm/charts/tree/master/stable/cluster-autoscaler).
+
+The AWS documentation for IRSA is here: https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html
+
+## Setup
+
+Run Terraform:
+
+```
+terraform init
+terraform apply
+```
+
+Set kubectl context to the new cluster: `export KUBECONFIG=kubeconfig_test-eks-irsa`
+
+Check that there is a node that is `Ready`:
+
+```
+$ kubectl get nodes
+NAME                                       STATUS   ROLES    AGE     VERSION
+ip-10-0-2-190.us-west-2.compute.internal   Ready    <none>   6m39s   v1.14.8-eks-b8860f
+```
+
+Install Helm:
+
+```
+kubectl -n kube-system create serviceaccount tiller
+kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+helm init --service-account=tiller
```

Comment:
Maybe we can omit tiller installation. And add note that we must have a working helm installation. Because, it's not relevant here.

I personally use terraform with helm provider to perform cluster-autoscaler installation. But it doen't matter how we install it. What we need for the serviceaccount is here https://github.com/terraform-aws-modules/terraform-aws-eks/pull/710/files#diff-5d3e96c50e09c013f8553243cb0be4d6R3-R6

Reply:
I agree 100%. I just haven't used Helm 3 yet 😢

I'll redo next week with Helm 3. I need to it this anyway!

Reply:
> Maybe we can omit tiller installation. 

OK good input. I'll remove that part. But I'll keep the `helm install` command for the chart. Is it same for Helm 3?

Reply:
yep.